SwaprHQ/stackly-ui
# [SWA-206][FEAT] - Reset stackbox on network switch or stack creation

## Fixes: [SWA-210](https://linear.app/swaprhq/issue/STK-210/update-pipelines-and-build-process)

# Description
* Update `bun dev` commands per project (dApp and landing)
* Reset the stackbox on network switch or stack creation


# Visual Evidence 
https://www.loom.com/share/676905c320fe4b589458ecc91bb51e83?sid=9f247128-9179-41ce-a5cf-299c21b2892e


# How to test the changes
1) Pull this branch
2) Run the project locally
3) Go to Stackly dApp
4) Connect your wallet
5) Play around selecting strategies and then switching the network, or try creating a stack and wait for it to be successfully created. Any of these actions should reset the stackbox.
